EducationPortal.DAL/Identity/AccountManager.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using EducationPortal.DAL.Entities;
+using EducationPortal.DAL.FS;
+using EducationPortal.DAL.Infrastructure;
+
+namespace EducationPortal.DAL.Identity
+{
+    public class AccountManager
+    {
+        private IdentityContext db;
+
+        public AccountManager(IdentityContext context)
+        {
+            this.db = context;
+        }
+
+        public Account Authenticate(string login, string password)
+        {
+            Account account = db.Storage.Find<Account>(a => a.Login == login).FirstOrDefault();
+
+            if (account == null)
+            {
+                Console.WriteLine($"Account not found");
+                return default;
+            }
+
+            if (PasswordHasher.VerifyHashedPassword(account.Password, password))
+            {
+                return account;
+            }
+
+            return default;
+        }
+        
+        public async Task<Account> Register(string login, string password)
+        {
+            var exists = db.Storage.Find<Account>(a => a.Login == login).Any();
```

Comment:
add a new method Any to storage (repository)

Reply:
Fixed